src/backend/src/modules/pia-intake/jsonb-classes/review/mpo-review.ts

```diff
@@ -15,47 +15,63 @@ export const mpoReviewMetadata: Array<IFormField<MpoReview>> = [
     key: 'isAcknowledged',
     type: 'boolean',
     allowedUserTypesEdit: [UserTypesEnum.MPO],
+    isSystemGeneratedField: false,
```

Comment:
would it be ok to add a comment on what isSystemGeneratedField does or it's purpose ?

Reply:
Yeah, makes sense. Done. Added at the definition file - `form-field.interface.ts`